app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,37 @@
+<nav class="navbar navbar-expand-lg bg-body-secondary rounded mb-3 ">
+  <div class="container-fluid">
+    <%= link_to 'Home', recipes_path, class: 'navbar-brand' %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <%= link_to 'Public Recipes', public_recipes_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'All invetories', inventories_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'Foods', foods_path, class: 'nav-link' %>
+        </li>
```

Comment:
**[OPTIONAL]**
This button works perfectly, great job 👍 💯 
However, this is not one of the wireframes required. Also, we already have the `add food` button in the navigation so users can add the food directly so it's not necessary to display the `foods#index`.
You can remove this if you want, it's up to you